2-mandatory/2-financial-times.js

```diff
@@ -5,33 +5,69 @@
     Implement the function below, which will return a new array containing only article titles which will fit.
 */
 function potentialHeadlines(allArticleTitles) {
-    // TODO
+    const ARTICLE_TITLES = [];
+    for (let i = 0; i < allArticleTitles.length; i++) {
+       if (allArticleTitles[i].length <= 65) {
+        ARTICLE_TITLES.push(allArticleTitles[i]);
+       };
+       
+
+    }
+
+    return ARTICLE_TITLES;
 }
 
+
 /*
     The editor of the FT likes short headlines with only a few words!
     Implement the function below, which returns the title with the fewest words.
-    (you can assume words will always be seperated by a space)
+    (you can assume words will always be separated by a space)
 */
 function titleWithFewestWords(allArticleTitles) {
-    // TODO
+    let shortestTitle = "";
+    let shortestWordCount = Infinity;
+
+    for (let i = 0; i < allArticleTitles.length; i++) {
+        const words = allArticleTitles[i].split("");
+        const wordCount = words.length;
+        if (wordCount < shortestWordCount) {
+            shortestTitle = allArticleTitles[i];
+            shortestWordCount = wordCount;
+        }
+    }
+    return shortestTitle;
 }
 
 /*
     The editor of the FT has realised that headlines which have numbers in them get more clicks!
     Implement the function below to return a new array containing all the headlines which contain a number.
     (Hint: remember that you can also loop through the characters of a string if you need to)
 */
-function headlinesWithNumbers(allArticleTitles) {
-    // TODO
+function findingNumber(str) {
+  return /[0-9]/.test(str);
 }
 
-/*
```

Comment:
Good job 👍 
I like how you used another function to help you solve the problem 😄